src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php

```diff
@@ -101,6 +105,8 @@ public function generateEntityChangeRequest(
 
         if ($entity->hasComments()) {
             $payload['note'] = $entity->getComments();
+        } else {
+            $payload['note'] = self::DEFAULT_CHANGE_REQUEST_NOTE;
```

Comment:
Try to prevent `else` expressions at all costs. I think you can work around them here quite easily.

One thing to consider, this might be good candidate to extract the logic into a method?

Reply:
Business logic will be moved to ManageEntity.